### ISSUE TITLE
UselessParentheses false positive with clone

### ISSUE THREAD

**Issue by magnetik** [slevomat/coding-standard#562]
Hi,

The following code triggers an useless parenthese: 

`$newDate = (clone $object->getDate())->sub(new \DateInterval('PT5M'));`

Thanks !

**Comment by carusogabriel** [comment]
Reproducible with
```php
$e = (clone $f->foo())->bar($g);
```

but not with

```php
$e = (clone $f)->bar($g);
```

so maybe `clone` isn't the problem here :)

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/81a741717c92654da25c243f961c6775675a7827
